atomic_reactor/plugins/exit_koji_tag_build.py

```diff
@@ -0,0 +1,97 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import unicode_literals
+
+from atomic_reactor.constants import PLUGIN_KOJI_TAG_BUILD_KEY
+from atomic_reactor.koji_util import create_koji_session, TaskWatcher
+from atomic_reactor.plugin import ExitPlugin
+from atomic_reactor.plugins.exit_koji_promote import KojiPromotePlugin
+
+
+class KojiTagBuildPlugin(ExitPlugin):
```

Comment:
that way also if we would need change in that we don't have to modify it in 2 places